core/capabilities/ccip/launcher/deployment.go

```diff
@@ -32,17 +32,25 @@ type ccipDeployment struct {
 // Close shuts down all OCR instances in the deployment.
 func (c *ccipDeployment) Close() error {
 	var err error
+	// if this node doesn't have instances, it's already closed
+	if c == nil {
+		return nil
+	}
 
 	// shutdown active commit instance.
-	err = multierr.Append(err, c.commit.active.Close())
+	if c.commit.active != nil {
+		err = multierr.Append(err, c.commit.active.Close())
+	}
```

Comment:
Ah interesting nice find, active can be nil since we always start by setting the candidate.